spec/gem_example.rb

```diff
@@ -17,6 +17,19 @@
         expect(subject.downcase).to eq(gem.downcase)
       end
     end
-  end
 
+    describe "versions" do
+      it "assumes that future versions will be patched" do
+        unaffected_versions = advisory['unaffected_versions'] || []
+        patched_versions    = advisory['patched_versions'] || []
+
+        versions  = unaffected_versions + patched_versions
+
+        # If a gem is unpatched this test makes no sense
+        unless patched_versions.none?
+          expect(versions.any? { |version| version.match(/^>=|^>/)}).to be_truthy
```

Comment:
Good suggestion, I ended up having to sort the versions that the conditions applied to in order to make this work because the order is not consistent...